pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -483,7 +484,7 @@ func (o *MustGatherOptions) Run() error {
 			line := fmt.Sprintf("unable to parse image reference %s: %v", image, err)
 			o.log(line)
 			// ensure the errors bubble up to BackupGathering method for display
-			errs = []error{fmt.Errorf(line)}
+			errs = []error{stderrors.New(line)}
```

Comment:
I think the expectation is `errs = []error{fmt.Errorf("unable to parse image reference %s: %v", image, err)}`?. I don't think we need to use this package.